function-server/src/main/java/io/dispatchframework/javabaseimage/Entrypoint.java

```diff
@@ -11,11 +11,18 @@
  */
 public class Entrypoint {
 
+    public static volatile boolean healthy = true;
+
     public static void main(String[] args) throws Exception {
 
-        Class c = Class.forName(args[0]);
+        Class<?> c = Class.forName(args[0]);
+
+        int port = 8080;
+        if (System.getenv().containsKey("PORT")) {
+            port = Integer.valueOf(System.getenv().get("PORT"));
+        }
```

Comment:
Might be better to put the port resolution logic into the `POJFunctionServer` constructor instead of passing it into the constructor since it only seems to be used there.